### PR TITLE
Fixed all broken links and fixed the existing one

### PR DESCRIPTION
**Pull request by j0rt3g4**
Fixed all broken links and fixed the existing one to the minimal expression Issue #3158

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Teams/room-systems/with-office-365.md [hunk 1]
```diff
@@ -200,7 +200,7 @@ After you've completed the preceding steps to enable your Microsoft Teams Rooms
...
+5. In the **Assign licenses** section, you need to select Skype for Business Online (Plan 2) or Skype for Business Online (Plan 3), depending on your licensing and what you've decided in terms of needing Enterprise Voice. You'll have to use a Plan 3 license if you want to use Cloud PBX on Microsoft Teams Rooms. Minimally you will need CloudPBX for voice connectivity. Then configure hybrid voice or PSTN calling based on the PSTN connectivity method. See [Microsoft Teams Rooms licenses](skype-room-systems-v2.md) for more details.
```
Comment (JohanFreelancer9) [inline Teams/room-systems/with-office-365.md]:
```suggestion
5. In the **Assign licenses** section, you need to select Skype for Business Online (Plan 2) or Skype for Business Online (Plan 3), depending on your licensing and what you've decided in terms of needing Enterprise Voice. You'll have to use a Plan 3 license if you want to use Cloud PBX on Microsoft Teams Rooms. Minimally you will need Cloud PBX for voice connectivity. Then configure hybrid voice or PSTN calling based on the PSTN connectivity method. See [Microsoft Teams Rooms licenses](skype-room-systems-v2.md) for more details.
```

**Comment by j0rt3g4** [comment]
Hello @LanaChin  @JohanFreelancer9 
is space still missing or did you add it @LanaChin  ?

**Pull request merged**